addon/components/inputs/abstract-input.js

```diff
@@ -156,6 +156,12 @@ export default Component.extend(HookMixin, PropTypeMixin, {
     return required && valueEmpty
   },
 
+  @readOnly
+  @computed('cellConfig')
+  alwaysShowRequiredLabel (cellConfig) {
```

Comment:
Why was this necessary?

Reply:
I thought I needed that but actually don't so I'll remove it